src/Login/Login.jsx

```diff
@@ -68,7 +78,7 @@ class Login extends React.Component {
           <hr className='login-page__separator' />
           <div className='body-typo'>{this.props.data.text}</div>
           {
-            this.props.providers.map(
+            this.props.providers && this.props.providers.map(
```

Comment:
is this necessary to add if you added default props?

Reply:
Oh I changed the code yesterday and forgot to commit... let me update that! thanks